test/Microsoft.AspNetCore.Mvc.Core.Test/ApplicationModel/AttributeRouteModelTests.cs

```diff
@@ -175,6 +175,32 @@ public void ReplaceTokens_InvalidFormat(string template, Dictionary<string, stri
         }
 
         [Fact]
+        public void ReplaceTokens_UnknownValue_InConstraints()
```

Comment:
Instead make the existing `ReplaceTokens_UnknownValue()` a `[Theory]`. You've already made one of the main required changes there.